### PR TITLE
55 use pretty time

### LINKED ISSUE #55: Use PrettyTime library

### PR DESCRIPTION
**Pull request by joeleba**
Fixes #55 
- Using Pretty Time
- Updated date parsing methods
- Added some custom error messages

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a floating task to the list. "
-            + "Parameters: TITLE [t/TAG1,TAG2...] [desc/DESCRIPTION]\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to the list. "
+            + "Parameters: TITLE [dl/DATE TIME][t/TAG1,TAG2...] [desc/DESCRIPTION]\n"
```
Comment (tjonganthony) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: Can you pull first. This has been updated to only use start and end option.

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -27,8 +27,8 @@
     public static final String DEADLINE_FLAG = "dl/";
     public static final String TAG_FLAG = "#";
     public static final String DESC_FLAG = "desc/";
-    public static final String WRONG_DATE_TIME_INPUT = "Wrong date format supplied. The correct format is YYYY-MM-DD HH:mm";
-    public static final Pattern DATE_TIME_FORMAT = Pattern.compile("\\d{4}-\\d{1,2}-\\d{1,2} \\d{2}:\\d{2}");
+    public static final String WRONG_DATE_TIME_INPUT = "I'm confused. My best guesses: %s";
+    public static final String UNABLE_TO_PARSE_DATE_TIME_INPUT = "I'm confused. What do you mean by %s?";
```
Comment (tjonganthony) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: "Date Time not recognised, try XXXXXX"?

**Comment by tjonganthony** [comment]
Looks good in general. Do look at my suggestion for the message though.
Maybe I'll need to take a look after merging into master to patch in Event and fix some logic. Ping me when you have merged this into master.

**Comment by joeleba** [comment]
Resolved merge conflicts. Applied @tjonganthony's suggestions.

**Pull request merged**